### ISSUE TITLE
LCIA not updated

### ISSUE THREAD

**Issue by LTFSie** [LCA-ActivityBrowser/activity-browser#1013]
I couldn’t find this issue raised on github so far. The latest ecoinvent version should come with the latest impact assessment methods (ReCiPe2016, IPCC2021, etc.). I reinstalled the ab (also deleted all databases and environments etc.), updated the biosphere and updated/loaded the latest ecoinvent version, but still only see the previous LCI methods. What am I doing wrong? Can you help me with this?

below is my conda env list output:
```
# Name                    Version                   Build  Channel
activity-browser          2.7.4                      py_0    bsteubing
appdirs                   1.4.4              pyh9f0ad1d_0    conda-forge
arrow                     1.2.3              pyhd8ed1ab_0    conda-forge
asteval                   0.9.31             pyhd8ed1ab_0    conda-forge
astunparse                1.6.3              pyhd8ed1ab_0    conda-forge
beautifulsoup4            4.12.2             pyha770c72_0    conda-forge
blinker                   1.6.2              pyhd8ed1ab_0    conda-forge
brightway2                2.4.3              pyhd8ed1ab_0    conda-forge
brotli                    1.0.9                hcfcfb64_9    conda-forge
brotli-bin                1.0.9                hcfcfb64_9    conda-forge
brotli-python             1.0.9            py39h99910a6_9    conda-forge
brotlicffi                1.0.9.2          py39h99910a6_3    conda-forge
bw2analyzer               0.10               pyhd8ed1ab_0    conda-forge
bw2calc                   1.8.2            py39hcbf5309_0    conda-forge
bw2data                   3.6.6              pyhd8ed1ab_0    conda-forge
bw2io                     0.8.8              pyhd8ed1ab_1    conda-forge
bw2parameters             1.1.0              pyhd8ed1ab_0    conda-forge
bw_migrations             0.2                pyhd8ed1ab_0    conda-forge
bzip2                     1.0.8                h8ffe710_4    conda-forge
ca-certificates           2023.7.22            h56e8100_0    conda-forge
certifi                   2023.7.22          pyhd8ed1ab_0    conda-forge
cffi                      1.15.1           py39h68f70e3_3    conda-forge
charset-normalizer        3.2.0              pyhd8ed1ab_0    conda-forge
click                     8.1.6           win_pyh7428d3b_0    conda-forge
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
contourpy                 1.1.0            py39h1f6ef14_0    conda-forge
cycler                    0.11.0             pyhd8ed1ab_0    conda-forge
dill                      0.3.7              pyhd8ed1ab_0    conda-forge
docopt                    0.6.2                      py_1    conda-forge
eidl                      1.5.1              pyhd8ed1ab_1    conda-forge
eight                     1.0.1              pyhd8ed1ab_4    conda-forge
et_xmlfile                1.1.0              pyhd8ed1ab_0    conda-forge
fasteners                 0.17.3             pyhd8ed1ab_0    conda-forge
flask                     2.3.2              pyhd8ed1ab_0    conda-forge
fonttools                 4.42.0           py39ha55989b_0    conda-forge
freetype                  2.12.1               h546665d_1    conda-forge
future                    0.18.3             pyhd8ed1ab_0    conda-forge
gettext                   0.21.1               h5728263_0    conda-forge
glib                      2.76.4               h12be248_0    conda-forge
glib-tools                2.76.4               h12be248_0    conda-forge
gst-plugins-base          1.22.5               h001b923_0    conda-forge
gstreamer                 1.22.5               hb4038d2_0    conda-forge
icu                       72.1                 h63175ca_0    conda-forge
idna                      3.4                pyhd8ed1ab_0    conda-forge
importlib-metadata        6.8.0              pyha770c72_0    conda-forge
importlib-resources       6.0.1              pyhd8ed1ab_0    conda-forge
importlib_metadata        6.8.0                hd8ed1ab_0    conda-forge
importlib_resources       6.0.1              pyhd8ed1ab_0    conda-forge
inflate64                 0.3.1            py39ha55989b_3    conda-forge
intel-openmp              2023.2.0         h57928b3_49496    conda-forge
itsdangerous              2.1.2              pyhd8ed1ab_0    conda-forge
jinja2                    3.1.2              pyhd8ed1ab_1    conda-forge
kiwisolver                1.4.4            py39h1f6ef14_1    conda-forge
krb5                      1.20.1               heb0366b_0    conda-forge
lcms2                     2.15                 h3e3b177_1    conda-forge
lerc                      4.0.0                h63175ca_0    conda-forge
libblas                   3.9.0              17_win64_mkl    conda-forge
libbrotlicommon           1.0.9                hcfcfb64_9    conda-forge
libbrotlidec              1.0.9                hcfcfb64_9    conda-forge
libbrotlienc              1.0.9                hcfcfb64_9    conda-forge
libcblas                  3.9.0              17_win64_mkl    conda-forge
libclang                  16.0.6          default_heb8d277_1    conda-forge
libclang13                16.0.6          default_hc80b9e7_1    conda-forge
libdeflate                1.18                 hcfcfb64_0    conda-forge
libffi                    3.4.2                h8ffe710_5    conda-forge
libglib                   2.76.4               he8f3873_0    conda-forge
libhwloc                  2.9.2           nocuda_h15da153_1008    conda-forge
libiconv                  1.17                 h8ffe710_0    conda-forge
libjpeg-turbo             2.1.5.1              hcfcfb64_0    conda-forge
liblapack                 3.9.0              17_win64_mkl    conda-forge
libogg                    1.3.4                h8ffe710_1    conda-forge
libpng                    1.6.39               h19919ed_0    conda-forge
libsqlite                 3.42.0               hcfcfb64_0    conda-forge
libtiff                   4.5.1                h6c8260b_0    conda-forge
libvorbis                 1.3.7                h0e60522_0    conda-forge
libwebp                   1.3.1                hcfcfb64_0    conda-forge
libwebp-base              1.3.1                hcfcfb64_0    conda-forge
libxcb                    1.15                 hcd874cb_0    conda-forge
libxml2                   2.11.5               hc3477c8_0    conda-forge
libxslt                   1.1.37               h6070c61_1    conda-forge
libzlib                   1.2.13               hcfcfb64_5    conda-forge
lxml                      4.9.3            py39hbae3653_0    conda-forge
m2w64-gcc-libgfortran     5.3.0                         6    conda-forge
m2w64-gcc-libs            5.3.0                         7    conda-forge
m2w64-gcc-libs-core       5.3.0                         7    conda-forge
m2w64-gmp                 6.1.0                         2    conda-forge
m2w64-libwinpthread-git   5.0.0.4634.697f757               2    conda-forge
markupsafe                2.1.3            py39ha55989b_0    conda-forge
matplotlib-base           3.7.1            py39haf65ace_0    conda-forge
mkl                       2022.1.0           h6a75c08_874    conda-forge
mrio_common_metadata      0.2.1              pyhd8ed1ab_0    conda-forge
msys2-conda-epoch         20160418                      1    conda-forge
multiprocess              0.70.15          py39ha55989b_0    conda-forge
multivolumefile           0.2.3              pyhd8ed1ab_0    conda-forge
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
networkx                  3.1                pyhd8ed1ab_0    conda-forge
numpy                     1.23.5           py39hbccbffa_0    conda-forge
openjpeg                  2.5.0                ha2aaf27_2    conda-forge
openpyxl                  3.1.2            py39ha55989b_0    conda-forge
openssl                   3.1.2                hcfcfb64_0    conda-forge
packaging                 23.1               pyhd8ed1ab_0    conda-forge
pandas                    2.0.3            py39h1679cfb_1    conda-forge
pathos                    0.3.1              pyhd8ed1ab_0    conda-forge
patsy                     0.5.3              pyhd8ed1ab_0    conda-forge
pcre2                     10.40                h17e33f8_0    conda-forge
peewee                    3.16.3           py39habf9222_0    conda-forge
pillow                    10.0.0           py39ha9166d5_0    conda-forge
pint                      0.21               pyhd8ed1ab_0    conda-forge
pip                       23.2.1             pyhd8ed1ab_0    conda-forge
platformdirs              3.10.0             pyhd8ed1ab_0    conda-forge
pooch                     1.7.0              pyha770c72_3    conda-forge
pox                       0.3.3              pyhd8ed1ab_0    conda-forge
ppft                      1.7.6.7            pyhd8ed1ab_0    conda-forge
psutil                    5.9.5            py39ha55989b_0    conda-forge
pthread-stubs             0.4               hcd874cb_1001    conda-forge
pthreads-win32            2.9.1                hfa6e2cd_3    conda-forge
py7zr                     0.20.6             pyhd8ed1ab_0    conda-forge
pybcj                     1.0.1            py39ha55989b_2    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pycryptodomex             3.18.0           py39ha55989b_0    conda-forge
pypardiso                 0.4.2              pyhd8ed1ab_1    conda-forge
pyparsing                 3.1.1              pyhd8ed1ab_0    conda-forge
pyperclip                 1.8.2              pyhd8ed1ab_2    conda-forge
pyppmd                    1.0.0            py39h99910a6_1    conda-forge
pyprind                   2.11.2          py39hcbf5309_1004    conda-forge
pyside2                   5.15.8           py39hdfd5cee_3    conda-forge
pysocks                   1.7.1              pyh0701188_6    conda-forge
python                    3.9.17          h4de0772_0_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-tzdata             2023.3             pyhd8ed1ab_0    conda-forge
python_abi                3.9                      3_cp39    conda-forge
pytz                      2023.3             pyhd8ed1ab_0    conda-forge
pyxlsb                    1.0.10             pyhd8ed1ab_0    conda-forge
pyzstd                    0.15.9           py39h95af829_0    conda-forge
qt-main                   5.15.8              h2c8576c_12    conda-forge
qt-webengine              5.15.8               h5b1ea0b_0    conda-forge
requests                  2.31.0             pyhd8ed1ab_0    conda-forge
salib                     1.4.7              pyhd8ed1ab_0    conda-forge
scipy                     1.11.1           py39hde5eda1_0    conda-forge
seaborn                   0.12.2               hd8ed1ab_0    conda-forge
seaborn-base              0.12.2             pyhd8ed1ab_0    conda-forge
setuptools                68.0.0             pyhd8ed1ab_0    conda-forge
setuptools-scm            7.1.0              pyhd8ed1ab_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
soupsieve                 2.3.2.post1        pyhd8ed1ab_0    conda-forge
stats_arrays              0.6.5              pyhd8ed1ab_0    conda-forge
statsmodels               0.14.0           py39hbaa61f9_1    conda-forge
tabulate                  0.9.0              pyhd8ed1ab_1    conda-forge
tbb                       2021.10.0            h91493d7_0    conda-forge
texttable                 1.6.7              pyhd8ed1ab_0    conda-forge
tk                        8.6.12               h8ffe710_0    conda-forge
tomli                     2.0.1              pyhd8ed1ab_0    conda-forge
typing-extensions         4.7.1                hd8ed1ab_0    conda-forge
typing_extensions         4.7.1              pyha770c72_0    conda-forge
tzdata                    2023c                h71feb2d_0    conda-forge
ucrt                      10.0.22621.0         h57928b3_0    conda-forge
unicodecsv                0.14.1                     py_1    conda-forge
unicodedata2              15.0.0           py39ha55989b_0    conda-forge
unidecode                 1.3.6              pyhd8ed1ab_0    conda-forge
urllib3                   2.0.4              pyhd8ed1ab_0    conda-forge
vc                        14.3                h64f974e_17    conda-forge
vc14_runtime              14.36.32532         hfdfe4a8_17    conda-forge
voluptuous                0.13.1             pyhd8ed1ab_0    conda-forge
vs2015_runtime            14.36.32532         h05e6639_17    conda-forge
werkzeug                  2.3.7              pyhd8ed1ab_0    conda-forge
wheel                     0.41.1             pyhd8ed1ab_0    conda-forge
whoosh                    2.7.4            py39hcbf5309_7    conda-forge
win_inet_pton             1.1.0              pyhd8ed1ab_6    conda-forge
wrapt                     1.15.0           py39ha55989b_0    conda-forge
xlrd                      2.0.1              pyhd8ed1ab_3    conda-forge
xlsxwriter                3.1.2              pyhd8ed1ab_0    conda-forge
xorg-libxau               1.0.11               hcd874cb_0    conda-forge
xorg-libxdmcp             1.1.3                hcd874cb_0    conda-forge
xz                        5.2.6                h8d14728_0    conda-forge
zipfile-deflate64         0.2.0            py39ha55989b_3    conda-forge
zipp                      3.16.2             pyhd8ed1ab_0    conda-forge
zstd                      1.5.2                h12be248_7    conda-forge
```

**Comment by marc-vdm** [comment]
Thanks for sharing with the `conda list` output!

Could you do the following:
`conda config --prepend channels conda-forge`
and then re-install activity-browser like normal?
 This should update AB to the right version (`2.8.x`), you can check if it worked with the following:
`conda list activity-browser`

If it didn't work, please let us know and we'll try and fix it another way!

**Comment by LTFSie** [comment]
Thanks @marc-vdm!
I get the following for conda config --prepend channels conda-forge:
Warning: 'conda-forge' already in 'channels' list, moving to the top

And for conda update activity-browser:
Collecting package metadata (current_repodata.json): done
Solving environment: done

# All requested packages already installed.

It seems the update did not work (conda list activity-browser):
# Name                    Version                   Build  Channel
activity-browser          2.7.4                      py_0    bsteubing

We might have to fix it another way?!

**Comment by marc-vdm** [comment]
Could you do the following `conda config --set channel_priority strict` and then try to install AB again?

If that doesn't work, could you post the output of `conda config --show channels`?

**Comment by LTFSie** [comment]
Thanks @marc-vdm! I was able to install the the ab version 2.8.0, but still can't see the latest LCIA methods. The 
/conda config --show channels
are:
  - conda-forge
  - defaults
  - cmutel
  - bsteubing

**Comment by marc-vdm** [comment]
Hmm did you also download ecoinvent 3.9 (or 3.9.1) and then update the biosphere? I hope that would fix the issue

**Comment by LTFSie** [comment]
Yes, several times. Unfortunately, no update in the Impact Categories.

**Comment by marc-vdm** [comment]
Could you share the new output of `conda list` again?

**Comment by LTFSie** [comment]
Sorry for the delayed answer. In a desperate moment I uninstalled everything, even miniconda3, which cashed some internal paths which I manually deleted in the end. So, I can't post my original conda list anymore... Reinstalling miniconda was also not possible. I ended up installing the less compact version anaconda - which finally worked!

While we can't present an elegant solution for my first issue, the more drastic version of reinstalling everything does do the trick. AB is up to date and also the Impact Categories!

Thank you so much for your support!
